memory-viz/src/automate.ts

```diff
@@ -189,20 +189,20 @@ function drawAutomatedOtherItems(
     const START_X = sf_endpoint + PADDING;
 
     for (const item of objs) {
-        if (
-            item.type === ".blank" &&
-            (item.width === undefined || item.height === undefined)
-        ) {
-            console.warn(
-                "WARNING :: An object with type='.blank' or '.blank-frame' exists with missing dimension information " +
-                    "(either the width or the height is missing). This object will be omitted in the memory model" +
-                    " diagram."
-            );
-        }
-        if (item.type !== ".blank") {
+        if (item.type === ".blank") {
+            if (item.width === undefined || item.height === undefined) {
+                console.warn(
+                    "WARNING :: An object with type='.blank' or '.blank-frame' exists with missing dimension information " +
+                        "(either the width or the height is missing). This object will be omitted in the memory model" +
+                        " diagram."
+                );
+            }
+            item.layoutHeight = item.height;
```

Comment:
Overall, we should not have to use a separate `layoutHeight` attribute.

Instead, if you need to override the user-provided width and height, then overwrite the `height` and `width` fields directly.

Also, I don't think it makes sense to set this attributes for `.blank` objects, since the whole point is they're supposed to be skipped in the diagram if their width/height is not provided.